specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,48 @@
+StartInBackgroundMode
+===
+
+# Background
+By default, WebView2 instances get created with some priorities set higher, assuming the WV2 is being created in the foreground for an important purpose. However, not every developer wants to boost the WebView2 prelaunch phase at the cost of utilizing more system resources. The `StartInBackgroundMode` property specifies that whether WebView2 created with launch boost or not.
+
+
+# Description
+Setting `StartInBackgroundMode` to TRUE will signal WebView2 to start in a 'background mode' which will attempt to use less system resources during the creation and initial load. Default is FALSE.
+
+# Examples
+## Win32 C++
+```cpp
+auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+options->put_StartInBackgroundMode(TRUE));
+HRESULT hr = CreateCoreWebView2EnvironmentWithOptions(
+  nullptr, m_userDataFolder.c_str(), options.Get(),
+  Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
+     this, &AppWindow::OnCreateEnvironmentCompleted).Get());
+```
+## WinRT and .NET
+```c#
+auto options = new CoreWebView2EnvironmentOptions();
+options.StartInBackgroundMode = true;
+auto environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+```
+
+
+# API Details
+## Win32 C++
+```cpp
+interface ICoreWebView2StagingEnvironmentOptions3 : IUnknown 
+{
+  /// Setting `StartInBackgroundMode` to `TRUE` will signal WebView2 to start in a 'background mode'
+  /// which will attempt to use less system resources during the creation and initial load.

```

Comment:
'Creation' and 'initial load' are ambiguous - we're not saying creation of what and load is also ambiguous maybe talking about the first navigation? The documentation should make it clear when this behavior will take place in terms of WebView2 APIs or other documented WebView2 things. Is this low resource behavior during each CoreWebView2 creation and ending when the async method returns the controller to the caller? Or is it maybe only when starting the browser process?

Reply:
This will function on each WV2 process creation not limited on manager/browser process only. 
> 'Creation' and 'initial load' are ambiguous

Yep, I'll rephase it!

Reply:
I should make it more clear here. This new property works for disabling launch boost for all processes. For launch boost, we currently only use it for Manager process,  child processes like Utilities and Render is only for experimental purpose (under sandbox). 

It's a good call. I should clarify that on the doc.